game/game.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+require_once(dirname( __FILE__ , 2).'/hand/dealer/dealer.php');
+require_once(dirname( __FILE__ , 2).'/hand/player/player.php');
+require_once(dirname( __FILE__ , 2).'/decks/decks.php');
+require_once(dirname( __FILE__ , 2).'/methods/methods.php');
+
+class Game {
+  //アプリの実行のメソッド
+  public function execute(){
+    echo <<<EOM
+    ★☆★☆★☆★☆★☆★☆★☆★☆ブラックジャックへようこそ!★☆★☆★☆★☆★☆★☆★☆★☆
+    ゲームを開始します。
+    \n
+    EOM;
+
+    $decks = new Decks;
+
+    $decks->createDecks();
+
+    $Player = new Player;
```

Comment:
```suggestion
    $player = new Player();
```
変数名の先頭は原則大文字かと思います。また、これは好みもあるかもしれませんが初期化の際のカッコは書いてあげた方がいい気がします（省略可能ですが）。